RescheduleAppointments/force-app/main/default/lwc/aaRebookingCalendar/aaRebookingCalendar.html

```diff
@@ -0,0 +1,252 @@
+<template>
+    
+    <template if:true = {isWeekView}>
+        <hr class="breakLine">
+        <lightning-layout horizontal-align="right">
+            <lightning-layout-item alignment-bump="right" >
+                <span>
+                    <lightning-button
+                        variant="base"
+                        label={currentMonthYearLabel}
+                        title={alternativeTextForMonthBtn} 
+                        icon-name="utility:down"
+                        class="slds-m-left_x-small monthButton"
+                        icon-position = "right"
+                        onclick={handleMonthViewButton}
+                    ></lightning-button>
+                </span>
+            </lightning-layout-item >
+
+            <lightning-layout-item>
+                <span>
+                    <lightning-button-icon
```

Comment:
There are many components in this project that generate Mobile SLDS errors for missing labels:

> Clickable images, Lightning button icons, and form elements require labels for mobile users. SLDS(4)

We should add labels to these controls.

![image](https://user-images.githubusercontent.com/755462/214726804-5c42ccb8-4927-4416-9e3d-48a0416a767d.png)